addons/calendar/static/src/xml/notification_calendar.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<template>
+
+<t t-name="CalendarNotification" t-extend="Notification">
+    <t t-jquery=".o_notification_title > t" t-operation="replace">
+        <span  t-attf-class="link2event eid_{{widget.eid}}">
+            <t t-esc="widget.title"/>
+        </span>
+    </t>
+    <t t-jquery=".o_notification_content" t-operation="after">
+        <div class="mt-2">
+            <button type="button" class="btn btn-primary link2showed oe_highlight oe_form oe_button"><span>OK</span></button>
+            <button type="button" class="btn btn-link link2event">Details</button>
+            <button type="button" class="btn btn-link link2recall">Snooze</button>
```

Comment:
That make a long time that I didnt see this button, are you sure it is always used?

Reply:
Oh yes it is :p 
Dev for the notification: 2h
Dev for making that particular notification still work: 5h :')